### PR TITLE
jbrowse-next (0.8.0)

### PR DESCRIPTION
**Pull request by erasche**
- [x] remove old emails that bounce
- [x] Fix metadata issues
    yeah... this is how it was supposed to work from the beginning. Oops. 
    auswahl_251
- [x] SPARQL
- [ ] support blast tabular data in 12 and 24 column variants
- [x] https://github.com/elsiklab/multibigwig
- [x] configurable chunk size

xref https://github.com/galaxyproject/tools-iuc/issues/508

### CODE REVIEW

**Comment by erasche** [comment]
Never mind, can't include multibigwig due to lack of license. (@cmdcolin no chance this could be resolved? I'd love to ship this plugin for use in the galaxy tool)

**Comment by cmdcolin** [comment]
It is mit license https://github.com/elsiklab/multibigwig/blob/master/bower.json :)

**Comment by erasche** [comment]
Oh, I missed that. Awesome, thanks so much!

**Comment by erasche** [comment]
TODO: "if you can see this text you need to whitelist this tool"

**Comment by abretaud** [comment]
Just pushed some changes:
- updated to latest 1.15.4 version (not yet 1.16 as it's not yet supported by apollo)
- reworked/fixed the sparql and rest track, it should work now

I worked with @xgaia to test sparql, it almost works, but the last error is probably not in this jbrowse galaxy tool (maybe jbrowse, maybe the sparql query, this kind of stuff is... complicated).

I'll need to work on including the jbrowse plugins inside the conda package since with newest jbrowse releases they need to be built with webpack...
I'd like to test indexing gff with tabix too...

**Comment by erasche** [comment]
> updated to latest 1.15.4 version (not yet 1.16 as it's not yet supported by apollo)

Would that be an issue for us? I'm assuming the data directory format doesn't change between 1.15 and 1.16, since only the data dir is sent we could probably upgrade 

> I'll need to work on including the jbrowse plugins inside the conda package since with newest jbrowse releases they need to be built with webpack...

urgh, that's... gross, ok. I really loved that we could load plugins from github/rawgit, that was super convenient.

**Comment by abretaud** [comment]
> > updated to latest 1.15.4 version (not yet 1.16 as it's not yet supported by apollo)
> 
> Would that be an issue for us? I'm assuming the data directory format doesn't change between 1.15 and 1.16, since only the data dir is sent we could probably upgrade
Yep, maybe it would work great, we can try. I don't know why needs to be done to integrate 1.16 in apollo.

> 
> > I'll need to work on including the jbrowse plugins inside the conda package since with newest jbrowse releases they need to be built with webpack...
> 
> urgh, that's... gross, ok. I really loved that we could load plugins from github/rawgit, that was super convenient.

I agree! I haven't found an issue for this on https://github.com/GMOD/jbrowse, we could open one maybe

**Comment by abretaud** [comment]
Ok, some plugins seem to work with the plugins built in the conda package
Looks like ComboTrackSelector is broken, I don't know why yet

**Comment by abretaud** [comment]
@erasche when you have time, I think it's ready for a review
I added the multibigwig plugin, it's a bit hacky, but it seems to work
Apart from the themes, all plugins should work

File: tools/jbrowse/readme.rst [hunk 1]
```diff
@@ -25,18 +25,31 @@ likely to change at any time! Beware. ;)
...
--  0.5.1 Support for contextual menus. Conda tests.
--  0.5 Update existing instances on disk. Index names. Support HTML tracks
-   instead of Canvas. Support default tracks. General JBrowse optinos
--  0.4 Support for dataset collections and customisation of tracks including
-   labelling, colours, styling. Added support for genetic code selection.
-   Fixed package installation recipe issues.
--  0.3 Added support for BigWig, etc.
--  0.2 Added support for BAM, Blast, VCF.
--  0.1 Initial public release.
+- 0.8 aka 1.16.1+galaxy0
```
Comment (erasche) [inline tools/jbrowse/readme.rst]:
```suggestion
- 1.16.1+galaxy0
```

firefox made crazy version number changes, we can too :)

**Comment by bgruening** [comment]
Merci @erasche and @abretaud!

**Comment by erasche** [comment]
@abretaud could we maybe use `exit_code`? I worry that someday we have some python warning that causes the tool to fail unexpectedly.

**Comment by bgruening** [comment]
Super cool. Thanks all!

**Pull request merged**